src/doc/rustc/src/platform-support/hermit.md

```diff
@@ -56,9 +56,9 @@ Rust does not yet ship pre-compiled artifacts for these targets.
 To compile for these targets, you will either need to build Rust with the targets enabled
 (see “Building the targets” above), or build your own copy of `core` by using `build-std` or similar.
 
-Building Rust programs can be done by following the tutorial in our starter application [rusty-demo].
+Building Rust programs can be done by following the tutorial in our starter application [hermit-rs-template].
```

Comment:
What is a "Rust program" here?

You mean application code that is compiled together with the Hermit OS into a single unikernel, yes? Or however you would phrase it. I think this could motivate clicking the link more strongly, as it's a tutorial about something quite exotic.

```suggestion
As all Hermit programs are unikernels, building a Rust program also requires including the operating system code. A guide for doing so is provided in our starter [hermit-rs-template].
```

Reply:
I mostly think it could afford to be a stronger nudge because I know that I've seen people _start_ trying to understand building for a target from these docs, for whatever reason.

Reply:
That's almost exactly how I described it to a friend. I'll go ahead and put this in. Thanks for the help! 🔥